<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>441</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="ReadCurrentTemp">
           <property name="text">
            <string>Read Current Temp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label1">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="SetTemp">
           <property name="text">
            <string>SetTemp</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Current Threshold:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="CurrentThreshold">
           <property name="text">
            <string>no current thresholdl</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Reset PORT</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QDial" name="dial">
           <property name="maximum">
            <number>35</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Set Value</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTemp_Control">
    <property name="title">
     <string>Temp Control</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuTemp_Control"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>pressed()</signal>
   <receiver>lcdNumber</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>pressed()</signal>
   <receiver>dial</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>pressed()</signal>
   <receiver>pushButton_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SetTemp</sender>
   <signal>pressed()</signal>
   <receiver>dial</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SetTemp</sender>
   <signal>pressed()</signal>
   <receiver>lcdNumber</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SetTemp</sender>
   <signal>pressed()</signal>
   <receiver>pushButton_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ReadCurrentTemp</sender>
   <signal>clicked()</signal>
   <receiver>label1</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>pressed()</signal>
   <receiver>CurrentThreshold</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>lineEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEdit</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
